validphys2/src/validphys/filters.py

```diff
@@ -327,11 +380,37 @@ def _filter_closure_data(filter_path, data, fakepdf, fakenoise, filterseed, data
 
         closure_data = make_level1_data(data, closure_data, filterseed, data_index, sep_mult)
 
-    # ====== write commondata and systype files ======#
-    if fakenoise:
+        if inconsistent_data_settings:
```

Comment:
```suggestion
        if inconsistent_data_settings is not None:
```

[ref](https://peps.python.org/pep-0008/#programming-recommendations)